src/transformers/modeling_utils.py

```diff
@@ -1802,6 +1803,11 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         offload_state_dict = kwargs.pop("offload_state_dict", None)
         subfolder = kwargs.pop("subfolder", "")
 
+        if trust_remote_code is not None:
```

Comment:
```suggestion
        if trust_remote_code is True:
```
Just in case we get some `False` that does nothing either.